contracts/market/template/MarketCore.sol

```diff
@@ -162,12 +158,14 @@ contract MarketCore is AccessControlledAndUpgradeableModifiers, IMarketCommon, I
         uint256 tokenSupply = IPoolToken(poolFixedConfig.token).totalSupply();
         uint256 price = uint256(poolValue).div(tokenSupply);
 
+        // as a precautionary measure - we pause minting immediately in this case.
+        if (price < 1e9) mintingPaused = true;
```

Comment:
Pause the market early if the price ever gets too low. Our simulations make us believe this will never even get close to happening even after hundereds or years, but good to have the check for prudence.

Then the team (who is still alive then ;) ) will have time to work out best solutions etc.